scripts/bundle-rhmi-operators.sh

```diff
@@ -28,17 +26,20 @@ UPGRADE_RHMI="${UPGRADE:-false}"
 VERSIONS="${BUNDLE_VERSIONS:-$LATEST_VERSION}"
 ROOT=$(pwd)
 INDEX_IMAGE=""
+CATALOGUE_SOURCE_INSTALL="${OC_INSTALL:-true}"
```

Comment:
```suggestion
CATALOG_SOURCE_INSTALL="${OC_INSTALL:-true}"
```

Just to align with the official name